src/AspNetCoreModuleV2/CommonLib/debugutil.cpp

```diff
@@ -103,6 +105,7 @@ void SetDebugFlags(const std::wstring &debugValue)
             if (value >= 3) DEBUG_FLAGS_VAR |= ASPNETCORE_DEBUG_FLAG_INFO;
             if (value >= 4) DEBUG_FLAGS_VAR |= ASPNETCORE_DEBUG_FLAG_CONSOLE;
             if (value >= 5) DEBUG_FLAGS_VAR |= ASPNETCORE_DEBUG_FLAG_FILE;
+            if (value >= 6) DEBUG_FLAGS_VAR |= ASPNETCORE_DEBUG_FLAG_EVENTLOG;
```

Comment:
I think EVENTLOG should be above console and file.

Reply:
Why?

Reply:
Event logs are more accessible for debugging on most platforms (azure). No other big reason besides that.

Reply:
My thinking was that eventlog is the most inappropriate place to dump debug logs. Also, I would like to deprecate numbers anyway.

Reply:
Yeah let's drop the numbers.